docs/install/first-run.md

```diff
@@ -40,14 +44,14 @@ anonymous usage information from running instances.
 
 The Administrator can opt-out of this feature.
 
-## Login
+## 5. Done, lets login
 
-Once you complete the setup, you will be able to login as the Administrator user
-you created in the first step.
+Once you complete the setup, you will be able to login as the Administrator
+user that you created in [Step 2](#2-create-an-administrator).
 
-## Next Steps
+## 6. Next Steps
 
-Once logged in, there are three remaining tasks the Administrator must complete
+There are three remaining tasks the Administrator must complete
 before projects can be created
 
 ### 1. Create a Project Type
```

Comment:
@hardillb Does your work in adding these steps to the wizard remove these lines? Or move them out of `first-run.md` at least?

Reply:
Not just yet, there is an outstanding bit of work in #1194 to add these in the setup wizard.

Reply:
Will be resolved via #1250